src/azure-cli/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+**Security**
+
+* Added new commands `az atp show` and `az atp update`
```

Comment:
Can you make it in one line and rearrange the position for your release note in alphabetical order

Reply:
Sure